client/src/app/app.routes.ts

```diff
@@ -4,35 +4,35 @@ import {SubmissionComponent} from './submission/submission.component';
-  {path: 'home', component: MyProjectsComponent, canActivate: [AuthGuardService]},
-  {path: 'submissions/list', component: SubmissionListComponent,  canActivate: [AuthGuardService]},
-  {path: 'submissions/:tab/upload', component: SubmissionComponent,  canActivate: [AuthGuardService]},
-  {path: 'submissions/detail/:id', component: SubmissionComponent,  canActivate: [AuthGuardService]},
-  {path: 'submissions/detail/:id/:tab', component: SubmissionComponent,  canActivate: [AuthGuardService]},
-  {path: 'submissions/detail', component: SubmissionComponent, canActivate: [AuthGuardService]},
+  {path: '', component: MyProjectsComponent, canActivate: [UserIsLoggedIn]},
+  {path: 'home', component: MyProjectsComponent, canActivate: [UserIsLoggedIn]},
+  {path: 'submissions/list', component: SubmissionListComponent,  canActivate: [UserIsLoggedIn, UserIsWrangler]},
+  {path: 'submissions/:tab/upload', component: SubmissionComponent,  canActivate: [UserIsLoggedIn, UserIsWrangler]},
+  {path: 'submissions/detail/:id', component: SubmissionComponent,  canActivate: [UserIsLoggedIn, UserIsWrangler]},
+  {path: 'submissions/detail/:id/:tab', component: SubmissionComponent,  canActivate: [UserIsLoggedIn, UserIsWrangler]},
 
-  {path: 'projects/detail/:projectUuid/submissions/:tab', component: SubmissionComponent, canActivate: [AuthGuardService]},
-  {path: 'projects/detail/:id', component: ProjectComponent, canActivate: [AuthGuardService]},
-  {path: 'projects/detail', component: ProjectComponent, canActivate: [AuthGuardService]},
-  {path: 'projects/all', component: AllProjectsComponent, canActivate: [AuthGuardService]},
+  {path: 'projects/detail/:id', component: ProjectComponent, canActivate: [UserIsLoggedIn, WranglerOrOwner]},
+  {path: 'projects/detail', component: ProjectComponent, canActivate: [UserIsLoggedIn, WranglerOrOwner]},
+  {path: 'projects/all', component: AllProjectsComponent, canActivate: [UserIsLoggedIn, UserIsWrangler]},
 
-  {path: 'projects', component: MyProjectsComponent, canActivate: [AuthGuardService]},
-  {path: 'projects/new', component: ProjectFormComponent, canActivate: [AuthGuardService]},
-  {path: 'projects/:uuid', component: ProjectFormComponent, canActivate: [AuthGuardService]},
-  {path: 'projects/:uuid/:tab', component: ProjectFormComponent, canActivate: [AuthGuardService]},
+  {path: 'projects', component: MyProjectsComponent, canActivate: [UserIsLoggedIn]},
+  {path: 'projects/new', component: ProjectFormComponent, canActivate: [UserIsLoggedIn]},
+  {path: 'projects/:uuid', component: ProjectFormComponent, canActivate: [UserIsLoggedIn, WranglerOrOwner]},
+  {path: 'projects/:uuid/:tab', component: ProjectFormComponent, canActivate: [UserIsLoggedIn, WranglerOrOwner]},
 
-  {path: 'registration', component: RegistrationComponent, canActivate: [AuthGuardService]},
+  {path: 'registration', component: RegistrationComponent, canActivate: [UserIsLoggedIn]},
```

Comment:
I think the check if user has ebi email was removed

Reply:
The check is present [user-is-ebi.service.ts](https://github.com/ebi-ait/ingest-ui/blob/79877056a85ff9ef0af337de5ee10d1f95750dfb/client/src/app/shared/services/user-is-ebi.service.ts) but unused.
Any pages with that check would be inaccessible to contributors, and so it's only wrangler pages that would need that check, but we have a more specific check for them already [user-is-wrangler.service.ts](https://github.com/ebi-ait/ingest-ui/blob/79877056a85ff9ef0af337de5ee10d1f95750dfb/client/src/app/shared/services/user-is-wrangler.service.ts).

Reply:
I see. Makes sense yeah. Thanks!